.gitignore

```diff
@@ -15,4 +15,5 @@ projects/
 
 # Other files
 .DS_Store
+.ruby-version
```

Comment:
_(non-blocking)_

When I used latest Ruby (3.3.1) locally, I got a compatibility error which I'm having trouble recreating now since I've already ran `bundle install` using an older compatible Ruby version and I don't know how to get back to a clean slate with 3.3.1. It might be good to enforce Ruby version somewhere here, but that should probably go in Gemfile. Not something to be done with this PR though.

Reply:
Yeah, I ran into some sort of bundler versioning thing when I used the newest Ruby, and I'm not really _super_ experienced with Ruby, so I ended up just downgrading to whatever Ruby 3.1 was in `rbenv install -l` and that worked :shrug: